x-pack/test/functional/apps/maps/embeddable/dashboard.js

```diff
@@ -111,5 +112,15 @@ export default function({ getPageObjects, getService }) {
       const afterRefreshTimerTimestamp = await getRequestTimestamp();
       expect(beforeRefreshTimerTimestamp).not.to.equal(afterRefreshTimerTimestamp);
     });
+
+    // see https://github.com/elastic/kibana/issues/61596 on why it is specific to maps
+    it("dashboard's back button should work", async () => {
```

Comment:
Nit: Would you mind changing the test name to something that includes the tested behavior? `should work` is very vague. How about something like "dashboard's back button should navigate to previous page"?